src/main/resources/templates/favorites.html

```diff
@@ -28,35 +28,10 @@
     <title>Home</title>
 </head>
 <body onload="startup()">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="/">TvTracker</a>
-            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarSupportedContent">
-                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="/">Browse</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/favorites">Favorites</a>
-                    </li>
-                </ul>
 
+<div th:replace="fragments/topnav :: navbar">
 
-                <div class="d-flex">
-                    <a class="nav-link" href="#">Login</a>
-                    <select id="mediaTypeDropdown">
-                        <option value="movie">Movie</option>
-                        <option value="series">Series</option>
-                    </select>
-                    <input id="searchTextBox" class="form-control me-2" type="text" placeholder="Search" aria-label="Search">
-                    <button id="searchBtn" class="btn btn-outline-success">Search</button>
-                </div>
-            </div>
-        </div>
-    </nav>
```

Comment:
I removed the navbar and put it into a file called topnav.html so that you could reuse it with minimal code using Thymeleaf as this is one of sonarcubes 7 deadly sins
https://docs.sonarqube.org/display/SONARQUBE45/Duplications

Reply:
Ah, good idea.  Re-usability.

Excellent source citation as well.